### PR TITLE
And fences highlight

### PR DESCRIPTION
**Pull request by williamdes**

### CODE REVIEW

**Comment by williamdes** [comment]
Hello @shalzz 
What do you think about this small change ?

**Comment by shalzz** [comment]
Hi, this looks like a more fine grained permissions control, which is fine of you want to have it in your own workshop.
In the example I want to keep it simple for people using the action.

It's reasonable for them to go through the settings to enable wrote permissions. Something people overlook is that you need to know how to use github actions before you try an actual action. I cannot replicate the whole GH actions manual/documentation here.

File: README.md [hunk 1]
```diff
@@ -41,34 +46,39 @@ jobs:
...
           BUILD_FLAGS: --drafts
           # A GitHub token is not necessary when BUILD_ONLY is true
           # GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          
+
   build_and_deploy:
     runs-on: ubuntu-latest
     if: github.ref == 'refs/heads/main'
+    permissions:
+      contents: write
```
Comment (williamdes) [inline README.md:81]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -41,34 +46,39 @@ jobs:
 
 This example will build and deploy to gh-pages branch on a push to the main branch, 
 and it will build only on pull requests.
-```
+```yml
 name: Zola on GitHub Pages
 
+permissions:
+  contents: read
+
```
Comment (williamdes) [inline README.md:54]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -30,6 +33,8 @@ jobs:
   build:
     name: Publish site
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
```
Comment (williamdes) [inline README.md:37]:
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -18,9 +18,12 @@ In your repository **Settings > Actions > General**, in Workflow permissions, ma
 
 This example will build and deploy to gh-pages on push to the main branch.
 
-```
+```yml
 name: Zola on GitHub Pages
 
+permissions:
+  contents: read
+
```
Comment (williamdes) [inline README.md:26]:
```suggestion
```

**Comment by williamdes** [comment]
> Hi, this looks like a more fine grained permissions control, which is fine if you want to have it in your own workflow. In the example I want to keep it simple for people using the action.
> 
> It's reasonable for them to go through the settings to enable write permissions. Something people overlook is that you need to know how to use github actions before you try an actual action. I cannot replicate the whole GH actions manual/documentation here.

Okay, I understand this. I removed the permissions changes, the PR should be okay now ?

**Review by shalzz — APPROVED** [review]

**Comment by shalzz** [comment]
Yes, thanks and merged.

**Pull request merged**